src/style.css

```diff
@@ -2,0 +1,46 @@
+body,
+html {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  margin-bottom: 0;
+}
+
+.star {
+  width: 24px;
+  height: 24px;
+  background-image: url('./images/star.svg');
+  background-size: cover;
+  margin-right: 8px;
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). You have a right margin for `.star`, which is okay if you're consistent with the spacing around these elements and do not add a left margin to the stars.